### DISCUSSION TITLE
Blog post about Embed Privacy featuring YouTube, WordPress.tv and Spotify embeds.

### DISCUSSION THREAD

**Discussion by oldrup** [epiphyt/embed-privacy#213] [Show and tell]
Well, I performed a pretty thorough test of Embed Privacy, if I may say so myself, and ended up procrastinating some very opinionated CSS for styling the Spotify embeds. 

The article, I ended up with, is loaded with embeds. The key takeaways are, that no tracking, cookies or data exchange takes place with 3rd parties, and **page size is down by a whopping 94%**, improving sustainability rating from grade F to grade A.

As long no embeds are actually clicked, of course.

https://oldrup.dk/en/more-speed-less-tracking-with-embed-privacy/

Take from it what you can. All CC BY-SA 4.0

**Comment by MatzeKitt** [comment]
I love your effort you’re putting in! Makes fun to see the results! 🤩
